server.js

```diff
@@ -19,9 +12,79 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+// Documentation added at default path
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.status(200).json({
+    resMessage: "Netflix movies",
+    endPointsArray: "/movies - Returns an array of all movies",
+    endPointsArrayQuery: "/movies ? type, director, year",
+    endPointsSingleItem: "/movies/id/:id - /movies/title/:title"
+  });
+});
+// Returns an array of all movies and adds the possibility to query filter type, director and year
+app.get("/movies", (req, res) => {
+  const { type, director, year } = req.query;
+  let movies = netflixData;
+
+  if (type) {
+    movies = movies.filter(movie => movie.type.toLowerCase() === type.toLowerCase());
+  }
+  if (director) {
+    movies = movies.filter(movie => movie.director.toLowerCase() === director.toLowerCase());
+  }
+  if (year) {
+    movies = movies.filter(movie => +movie.release_year === +year);
+  }
+
+  res.status(200).json({
```

Comment:
If I request this and the year doesnt exist i still get repsonse status 200 "ok". https://project-express-api-kwmzxb52za-lz.a.run.app/movies?year=2021. But I think most of our codes look like that